　surefireレポートには、Postgresの「SQLState=25P02」で失敗。「失敗したSQLトランザクション内である」ということらしい。処理的にfinallyで失敗した際の回収処理でDAO触っているので起きているようなので、例外を上書きしてしまったんだろうけど、次のhudsonからの実行で何事もなく成功してしまっていた。
　hudson君にテスト結果の履歴を持たせる設定をしていなかったので、その後surefireレポートは上書きされてしまった。痛い。
　その後何度か同じテストケースを実行するも再現せず。トランザクション失敗するような状況になかったので原因不明。同時並行で動いているものもなかったようだし、再起動するなどもしていないので復旧のきっかけもないし。
　とりあえず忘れないようメモだけしておく。
